[2025-04-06T12:00:03.961+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-06T12:00:03.973+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_api_dag.create_weather_table scheduled__2025-04-06T11:00:00+00:00 [queued]>
[2025-04-06T12:00:03.979+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_api_dag.create_weather_table scheduled__2025-04-06T11:00:00+00:00 [queued]>
[2025-04-06T12:00:03.979+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-04-06T12:00:03.988+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): create_weather_table> on 2025-04-06 11:00:00+00:00
[2025-04-06T12:00:03.999+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=240969) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-06T12:00:03.999+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_api_dag', 'create_weather_table', 'scheduled__2025-04-06T11:00:00+00:00', '--job-id', '738', '--raw', '--subdir', 'DAGS_FOLDER/weatherAPIDAG.py', '--cfg-path', '/tmp/tmp6w7lnbq3']
[2025-04-06T12:00:04.001+0000] {standard_task_runner.py:105} INFO - Job 738: Subtask create_weather_table
[2025-04-06T12:00:04.000+0000] {standard_task_runner.py:72} INFO - Started process 240980 to run task
[2025-04-06T12:00:04.035+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_api_dag.create_weather_table scheduled__2025-04-06T11:00:00+00:00 [running]> on host ab13fcc6a433
[2025-04-06T12:00:04.166+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_api_dag' AIRFLOW_CTX_TASK_ID='create_weather_table' AIRFLOW_CTX_EXECUTION_DATE='2025-04-06T11:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-06T11:00:00+00:00'
[2025-04-06T12:00:04.167+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-04-06T12:00:04.168+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-04-06T12:00:04.168+0000] {logging_mixin.py:190} INFO - Current task name:create_weather_table state:running start_date:2025-04-06 12:00:03.973222+00:00
[2025-04-06T12:00:04.168+0000] {logging_mixin.py:190} INFO - Dag name:weather_api_dag and current dag run status:running
[2025-04-06T12:00:04.168+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-06T12:00:04.181+0000] {base.py:84} INFO - Retrieving connection 'weather_***_conn'
[2025-04-06T12:00:04.192+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-06T12:00:04.203+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-06T12:00:04.204+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_api_dag, task_id=create_weather_table, run_id=scheduled__2025-04-06T11:00:00+00:00, execution_date=20250406T110000, start_date=20250406T120003, end_date=20250406T120004
[2025-04-06T12:00:04.211+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-04-06T12:00:04.211+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-04-06T12:00:04.211+0000] {logging_mixin.py:190} INFO - Dag name:weather_api_dag queued_at:2025-04-06 12:00:00.076553+00:00
[2025-04-06T12:00:04.211+0000] {logging_mixin.py:190} INFO - Task hostname:ab13fcc6a433 operator:PythonOperator
[2025-04-06T12:00:04.257+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-06T12:00:04.277+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-06T12:00:04.279+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
